tests/ignite/distributed/utils/test_serial.py

```diff
@@ -22,6 +22,8 @@ def test_no_distrib(capsys):
     assert idist.backend() is None
     if torch.cuda.is_available():
         assert idist.device().type == "cuda"
+    elif torch.backends.mps.is_available():
```

Comment:
we have to put a guard `_torch_version_le_112` here as we have also tests for older pytorch version where mps backend does not exist.

Reply:
Also, to run the test, you need to remove `@pytest.mark.skipif`